examples/preprocessing/plot_ica_from_raw.py

```diff
@@ -62,6 +62,7 @@
 
 show_picks = np.abs(scores).argsort()[::-1][:5]
 
+ica.plot_raw_components(raw, exclude=ecg_inds, title=title % 'ecg')
```

Comment:
I would like to leave it like this, it has been very useful for tracking bugs in the past.